governance/scripts/bridge/payFee.js

```diff
@@ -31,7 +31,7 @@ const fetchRelayerFee = async ({ originDomain, destinationDomain }) => {
 
 async function main({
   // TODO: pass this hash via cli
-  txId = '0x12d380bb7f995930872122033988524727a9f847687eede0b4e1fb2dcb8fce68',
+  txId = '0xaa9c5da11ccb270ce2760ddcd64f2be8d56702c7aeaa32ef8da1536e7e7e4e98',
```

Comment:
latest tx from that proposal https://www.tally.xyz/gov/unlock/proposal/5197867276108850754638452366106160587013724784768388781483332671637332902539